akka-stream/src/main/scala/akka/stream/impl/io/SimpleUdpSinkActor.scala

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.impl.io
+
+import java.net.InetSocketAddress
+
+import akka.actor.{ ActorLogging, ActorRef, Props }
+import akka.io.{ IO, Udp }
+import akka.stream.actor.ActorSubscriberMessage.{ OnComplete, OnError, OnNext }
+import akka.stream.actor.{ ActorSubscriber, RequestStrategy, WatermarkRequestStrategy, ZeroRequestStrategy }
+import akka.util.ByteString
+
+/** INTERNAL API */
+private[akka] object SimpleUdpSinkActor {
+  def props(target: InetSocketAddress): Props =
+    Props(new SimpleUdpSinkActor(target))
+}
+
+/** INTERNAL API */
+private[akka] class SimpleUdpSinkActor(target: InetSocketAddress) extends ActorSubscriber with ActorLogging {
+
+  private var currentRequestStrategy: RequestStrategy = ZeroRequestStrategy
+  override def requestStrategy = currentRequestStrategy
+
+  import context.system
+
+  IO(Udp) ! Udp.SimpleSender
+
+  override def receive: Receive = {
+    case Udp.SimpleSenderReady ⇒
+      currentRequestStrategy = WatermarkRequestStrategy(10) // TODO arbitrary value, how can we do better here (needs dropping to low level)
```

Comment:
it could use the max inputBuffer size from the `effectiveSettings` (OperationAttributes). That will be easier to grab when we have merged https://github.com/akka/akka/pull/17175